### DISCUSSION TITLE
Exception hook not properly working..

### DISCUSSION THREAD

**Discussion by BorisKM** [livewire/livewire#9054] [7. Bugs (archived)]
### Livewire version

v3.5.15

### Laravel version

v11.34.2

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. Create livewire component
2. Add model as property
3. Fill property with existing model entity
4. Add any action that do nothing
5. Delete model entity from the DB
6. Fire action, and you will see 404 popup

### Repository or Wirebox URL

-

### Additional information

The problem is inside `ModelSynth` class.
For example if i will replace it with custom class that just do `->first()` instead of `->firstOrFail()` in `hydrate` method, then everything fine.. But this is bad approach.. Lifecicle hook `exception` should work!

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by lrljoe** [comment]
ModelSynth does firstOrFail when hydrating, so that it will fire an Exception when this use case happens.

```
        $model = (new $class)->newQueryForRestoration($key)->useWritePdo()->firstOrFail();
```

What's not working with the exception hook 

Is it happening with a full page component, or a child component?

**Reply by BorisKM** [reply]
With full-page component.
